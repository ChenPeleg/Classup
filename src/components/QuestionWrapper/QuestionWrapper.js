import React from "react";
import styles from "./QuestionWrapper.module.scss";
const QuestionWrapper = (props) => (
  <div className={styles.questionWrapper}>
    <div className={styles.questionOptions}> {props.children}</div>
  </div>
);
export default QuestionWrapper;
