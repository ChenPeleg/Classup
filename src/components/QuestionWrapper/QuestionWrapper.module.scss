@import "../../assets/styles/scrollbar";
@import "../../assets/styles/base";
.questionScroller {
  position: relative;
  width: 70%;
  left: 15%;
  border-radius: 2px;
  box-shadow: 2px 2px 4px 2px rgba(2, 0, 36, 0.5);
  margin: 10px;
  padding: 5px;
  padding-bottom: 15px;
  overflow-y: auto;
  overflow-x: hidden;
  height: 450px;
  @include scrollbars(10px, rgba(12, 134, 196, 0.986899), #00d4ff);
  @include tablet {
    font-size: $Mobile_fontSize;
    width: 75%;
    left: 10%;
  }
  @include mobile {
    font-size: $Mobile_fontSize;
    width: 85%;
    left: 3%;
  }
}
.questionWrapper {
  position: relative;
  display: block;
}
