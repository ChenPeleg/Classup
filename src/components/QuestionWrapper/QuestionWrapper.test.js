import React from "react"
import { shallow } from 'enzyme'
import QuestionWrapper from './QuestionWrapper'

const wrap = shallow(<QuestionWrapper />)

it("QuestionWrapper renders correctly", () => {
    expect(wrap).toMatchSnapshot()
})