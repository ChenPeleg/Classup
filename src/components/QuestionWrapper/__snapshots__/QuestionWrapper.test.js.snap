// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionWrapper renders correctly 1`] = `
<div
  className="questionWrapper"
>
  <div
    className="questionScroller"
  >
     
  </div>
</div>
`;
